templates/client/html/catalog/detail/body.php

```diff
@@ -48,7 +48,27 @@
 
 
 ?>
-<?php if( isset( $this->detailProductItem ) ) : ?>
+<?php if( isset( $this->detailProductItem ) ) : 
+
+        // preselection of a variant product in a selection article
+        $preSelectVariantAttributes = [];
+        if ($_GET['productId'] && is_numeric($_GET['productId'])):
```

Comment:
Don't use `$_GET`, but `$this->param( 'productId' )` instead

Reply:
My url looks like this:
`http://aimeos.local/detail/d/Testproduct-Selection?productId=19963`
and there is no $this->param( 'productId' ) in body.php.
I think I have to configure a url-param like
`http://aimeos.local/detail/d/Testproduct-Selection/0/19963`

Please can you assist me where I have to do that?

Reply:
Ok, I have foud it ;-)
https://aimeos.org/docs/2022.x/typo3/setup/#seo-urls